tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -173,58 +173,49 @@ install_initrd() {
 }
 
 #Install kernel asset
-install_kernel() {
-	export kernel_version="$(yq r $versions_yaml assets.kernel.version)"
+install_kernel_helper() {
+	local kernel_version_yaml_path="${1}"
+	local kernel_name="${2}"
+	local extra_cmd=${3}
+
+	export kernel_version="$(yq r $versions_yaml ${kernel_version_yaml_path})"
 	local kernel_kata_config_version="$(cat ${repo_root_dir}/tools/packaging/kernel/kata_config_version)"
```

Comment:
Just checking that it will use the same `kata_config_version` for all the kernel flavors, otherwise this `kernel_kata_config_version` should be parameterized as well.

Reply:
If we bump a config for any kernel we need to bump this file, so this is the same for all flavours.